### ISSUE TITLE
Some nav items lose keyboard focus

### ISSUE THREAD

**Issue by tsidel** [ThatConference/that.us#1066]
**Describe the bug**
When the top nav is on a white background, the "user" icon loses keyboard focus.
When the top nav is on a blue background (interior pages) the "activity" icon and user icon loses keyboard focus

**To Reproduce**
1. Go to https://that.us/
2. Click in the nav area (but not on anything)
3. Tab with your keyboard across the top nav items. You will see each navigation element that you could visit (by hitting enter or the spacebar) outlined in either white or blue.
4. On the front page, the user icon gets a flash of an orange border, then the border disappears. On interior pages (or navs on blue backgrounds) the border is not present when you tab to the "activity" or "user" icons

**Expected behavior**
Every navigation element should have a consistent visible indicator that is the active element. 

**Additional context**
Fixing this would meet WCAG 2.1 AA Success Criterion 2.4.7 Focus Visible (https://www.w3.org/TR/WCAG21/#focus-visible) "Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible."

**Comment by csell5** [comment]
ok, so I think this is part of how svelte is loading, or really the page might be loading. are you seeing it lose mouse focus too? meaning there is a brief moment where it seems dead.  all in all... I think there is more going on here than I know about. but it's pretty consistent on first-page load.

**Comment by tsidel** [comment]
I've been able to fix some of them by removing some of the tailwind(?) css classes. There are some that had a class of focus:outline-none, just removing that makes the focus appear as needed. The missing focus is also noticeable in the "user" icon and the "activity" icon on other pages.  So far it's been pretty easy to fix as soon as I find what's controlling what.

For the footer links, it'll become a design question of how do you want the focus to appear since it's on a white background. We could make the focus in the footer behave  like the front page nav and add focus:bg-that-blue, focus:white-text to the links but that seems a little much for a footer element. Just a border could work. 

The goal here is to give some visual clue to keyboard users that they are on an actionable element.. could be done with a background color change, or an outline, or even triggering an underline or border-bottom.

**Comment by tsidel** [comment]
This seems to be fixed now.
